dashboard/config/callouts.tsv

```diff
@@ -1,18 +1,18 @@
 Name	script_id	level_num	game_name	localization_key	element_id	qtip_config
-Drag Blocks 20	1	2_2	Maze	drag_blocks	[block-id='1']	{"position": {"my": "top left", "at": "bottom right", "adjust": {"x": 22, "y":60}}, "style": { "classes": "no-tip" }}
-Drag Blocks HoC	2	2_2	Maze	drag_blocks	[block-id='1']	{"position": {"my": "top left", "at": "bottom right", "adjust": {"x": 22, "y":60}}, "style": { "classes": "no-tip" }}
-Run 20	1	2_2	Maze	run	#runButton
-Run HoC	2	2_2	Maze	run	#runButton
-Watch Video Again 20	1	2_5	Maze	watch_again	[data-key='loop_times']	{"position": {"my": "bottom left", "at": "top right"}}
-Watch Video Again HoC	2	2_5	Maze	watch_again	[data-key='loop_times']	{"position": {"my": "bottom left", "at": "top right"}}
-Undraggable 20	1	2_8	Maze	gray_block_explanation	.blocklyUndeletable	{"position": {"my": "top middle", "at": "bottom middle"}}
-Undraggable HoC	2	2_8	Maze	gray_block_explanation	.blocklyUndeletable	{"position": {"my": "top middle", "at": "bottom middle"}}
-Show Code 20	1	2_13	Maze	show_code_explanation	#show-code-header	{"position": {"my": "center right", "at": "center left"}}
-Show Code HoC	2	2_13	Maze	show_code_explanation	#show-code-header	{"position": {"my": "center right", "at": "center left"}}
-Instructions 20	1	2_14	Maze	instructions_repeat_explanation	#prompt	{"position": {"my": "bottom left", "at": "top center"}}
-Instructions HoC	2	2_14	Maze	instructions_repeat_explanation	#prompt	{"position": {"my": "bottom left", "at": "top center"}}
-Category 20	1	2_1	Artist3	category_explanation	.blocklyTreeRoot	{"position": {"my": "center left", "at": "center right"}}
-Jigsaw Click	7	1	Jigsaw	jigsaw_click	[block-id='1']	{"position": {"my": "top left", "at": "center right"}}
+Drag Blocks 20	20-hour	2_2	Maze	drag_blocks	[block-id='1']	{"position": {"my": "top left", "at": "bottom right", "adjust": {"x": 22, "y":60}}, "style": { "classes": "no-tip" }}
```

Comment:
Unfortunately, the logic that checks if it's an id or a name will think this is a name:

https://github.com/code-dot-org/code-dot-org/blob/8ce9da9d38c97c2e2d7287ca9a42dc97fe608d2f/dashboard/app/models/callout.rb#L44

```
[development] dashboard > '20-hour'.to_i
=> 20
```

So I think we need to change the seeding logic too. We support id-or-name for scripts [here](https://github.com/code-dot-org/code-dot-org/blob/8ce9da9d38c97c2e2d7287ca9a42dc97fe608d2f/dashboard/app/models/script.rb#L564) -- I wonder if we can either use that function or steal some of its logic

Reply:
i can look into stealing some logic in a couple minutes!

Reply:
I'm also seeing errors related to iceage when I try to seed locally

Reply:
When I query `select * from levels where level_num = 'iceage_move_to_actor';`. Is it possible the `level_num` for these levels is incorrect?

Reply:
you're probably right, maureen. it seems unrelated, so i don't think it needs to block this change. iceage isn't very widely used, so fixing those callouts can be a follow-up task, especially since we aren't changing any iceage-related callouts in this PR

Reply:
Thanks for double checking! I'm feeling a bit paranoid right now as my "simple" change was the one that blew up the pipeline, so really appreciate the confidence check.